src/services/MakeStreamZip.php

```diff
@@ -134,12 +140,24 @@ private function addTimestampFiles(int $id): void
      * @return void
      */
     private function nameFolder(): void
```

Comment:
this function can be suppressed. It adds nothing and is only used once in this class.

Reply:
That's true, I removed the method and also the call to it in `addToZip`